src/storage/rocksdb.cpp

```diff
@@ -25,7 +27,7 @@ bool FInternalKey(const char *key, size_t cch)
 std::string getPrefix(unsigned int hashslot)
 {
     char *hash_char = (char *)&hashslot;
-    return std::string(hash_char + (sizeof(unsigned int) - 2), 2);
+    return std::string(hash_char, sizeof(unsigned int));
```

Comment:
You want uint16_t not unsigned, otherwise your wasting 2 bytes for every key

Reply:
updated